src/tree/cosmosdb/CosmosDBItemResourceItem.ts

```diff
@@ -60,33 +58,85 @@ export abstract class CosmosDBItemResourceItem
+        const id: string = (doc.id as string | undefined) ?? (doc._id as string | undefined) ?? '<no id>';
 
-    protected generatePartitionKeyTooltip(): string {
-        if (!this.model.container.partitionKey || this.model.container.partitionKey.paths.length === 0) {
-            return '';
+        const pkPaths = this.model.container.partitionKey?.paths ?? [];
+        const pkValuesRaw = this.getPartitionKeyValuesArray(this.model);
+
+        const lines: string[] = [];
+
+        // ID section
+        lines.push(`**ID:** ${id}`);
+
+        // Partition Key section
+        if (pkPaths.length > 0) {
+            lines.push('');
+
+            if (pkPaths.length === 1) {
+                const path = pkPaths[0];
+                const raw = pkValuesRaw[0];
+                lines.push('#### Partition Key');
+                lines.push(`  \`${path}\`: ${this.formatValue(raw)}`);
+            } else {
+                lines.push(``);
+                lines.push(`| Partition Key | Value |`);
+                lines.push(`|---|---|`);
+                for (let i = 0; i < pkPaths.length; i++) {
+                    const path = pkPaths[i];
+                    const raw = pkValuesRaw[i];
+                    lines.push(`|${path}|${this.formatValue(raw)}|`);
+                }
+            }
         }
 
-        const partitionKeyPaths = this.model.container.partitionKey.paths.join(', ');
-        const partitionKeyValues = this.generatePartitionKeyValue(this.model);
+        return lines.join('\n');
+    }
 
-        return (
-            '### Partition Key\n' +
-            '---\n' +
-            `- Paths: **${partitionKeyPaths}**\n` +
-            `- Values: **${partitionKeyValues}**\n`
-        );
+    private getPartitionKeyValuesArray(model: CosmosDBItemModel): unknown[] {
+        if (!model.container.partitionKey || model.container.partitionKey.paths.length === 0) {
+            return [];
+        }
+        const partitionKeyValues = extractPartitionKey(model.item, model.container.partitionKey);
+        return Array.isArray(partitionKeyValues) ? partitionKeyValues : [partitionKeyValues];
+    }
+
+    private formatValue(value: unknown): string {
+        if (value === null) {
+            return '`null`';
+        }
+        if (value === undefined) {
+            return '`undefined`';
+        }
+        if (typeof value === 'object') {
+            try {
+                const json = JSON.stringify(value);
+                // Truncate long objects
+                if (json.length > 100) {
+                    return '`' + json.substring(0, 97) + '…`';
+                }
+                return '`' + json + '`';
+            } catch {
+                return '`[object]`';
+            }
+        }
+        if (typeof value === 'string') {
+            // Truncate long strings
+            if (value.length > 100) {
+                return `"${value.substring(0, 97)}…"`;
+            }
+            return `"${value}"`;
```

Comment:
I don't remember how it works in JS, but I'm 95% sure that here the user will see "value", but in fields above it will be \`value\`